src/Mailman.Server/ClientApp/src/components/MergeTemplate/MUIReactAutosuggest.js

```diff
@@ -0,0 +1,259 @@
+// Source: https://material-ui.com/demos/autocomplete/
```

Comment:
This file should be renamed MuiReactAutosuggest 
(lowercase U and I in the filename)